# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/moz.build
# Commit: b76bd797f69d
# Full Hash: b76bd797f69dd1ee1336807f6c872ad7b1b2b656
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-03-06 21:16:27
# Regressor Bug: 1858958
# File Overlap Count: 1
# Description:
#   Bug 1858958 - WebIDL file and skeleton code for AudioData. r=chunmin,webidl,saschanaz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D190943
# ==============================================================================

diff -r 840e1a0d6c22 -r b76bd797f69d dom/media/webcodecs/moz.build
--- a/dom/media/webcodecs/moz.build	Wed Mar 06 10:29:52 2024 +0000
+++ b/dom/media/webcodecs/moz.build	Wed Mar 06 11:14:15 2024 +0000
@@ -21,6 +21,7 @@
 ]
 
 EXPORTS.mozilla.dom += [
+    "AudioData.h",
     "DecoderTemplate.h",
     "DecoderTypes.h",
     "EncodedVideoChunk.h",
@@ -35,6 +36,7 @@
 ]
 
 UNIFIED_SOURCES += [
+    "AudioData.cpp",
     "DecoderAgent.cpp",
     "DecoderTemplate.cpp",
     "EncodedVideoChunk.cpp",